### DISCUSSION TITLE
NIC uplink delay

### DISCUSSION THREAD

**Discussion by rikmm** [ltsp/ltsp#724] [General]
Hi,

I know it's not directly an LTSP problem, but i'm trying to speed up the boot process.

Everything is running fast except the NIC that takes about 8 seconds to be up.

I use LTSP 22.01-2, with a Ubuntu 22.04 and the nic driver is e1000e.

I guess it's the auto negotiation that delays the boot.

Is there something i could try ?

Cheers
Eric

**Comment by alkisg** [comment]
Hi Eric,

a video of the boot process might help to pinpoint where the problem is.
Additionally, you may test with different clients and different switches, to see if the problem is specific to e1000e or even specific to this switch.
E.g. I think I've heard such an issue with some cisco switches, and the solution was to disable the spanning tree protocol on the client ports.
I haven't tested myself though, I only have cheap unmanaged switches here! :D
